Application/EdFi.Ods.AdminApi/Infrastructure/Security/SecurityExtensions.cs

```diff
@@ -1,110 +1,110 @@
-// SPDX-License-Identifier: Apache-2.0
-// Licensed to the Ed-Fi Alliance under one or more agreements.
-// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-// See the LICENSE and NOTICES files in the project root for more information.
-
-using EdFi.Ods.AdminApi.Features.Connect;
-using Microsoft.AspNetCore.Authentication.JwtBearer;
+// SPDX-License-Identifier: Apache-2.0

```

Comment:
Sorry about the diff. Apparently line endings were wrong on this file.

Reply:
np